examples/complete/variables.tf

```diff
@@ -69,3 +69,23 @@ variable "service_credential_names" {
     "postgressql_editor" : "Editor",
   }
 }
+
+variable "configuration" {
+  description = "Database configuration"
+  type = object({
+    max_connections            = optional(number)
+    max_prepared_transactions  = optional(number)
+    deadlock_timeout           = optional(number)
+    effective_io_concurrency   = optional(number)
+    max_replication_slots      = optional(number)
+    max_wal_senders            = optional(number)
+    shared_buffers             = optional(number)
+    synchronous_commit         = optional(string)
+    wal_level                  = optional(string)
+    archive_timeout            = optional(number)
+    log_min_duration_statement = optional(number)
+  })
+  default = {
+    max_connections = 250
+  }
+}
```

Comment:
Can you just add this directly in the the example main.tf. Also update the example readme with a one liner saying the max connection value is being set?